### DISCUSSION TITLE
Livewire datatable does not work in laravel 10

### DISCUSSION THREAD

**Discussion by maya1987** [livewire/livewire#5602] [1. Help]
I am working on a project in laravel 10.0.I failed to install  mediconesystems/livewire-datatables  .It shows some compatibility issues. The screenshot for the error pasted here

`./composer.json has been updated
Running composer update mediconesystems/livewire-datatables
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires mediconesystems/livewire-datatables ^0.10.0 -> satisfiable by mediconesystems/livewire-datatables[v0.10.0].
    - mediconesystems/livewire-datatables v0.10.0 requires illuminate/support ^7.0|^8.0|^9.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.4] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require mediconesystems/livewire-datatables:*" to figure out if any version is installable, or "composer require mediconesystems/livewire-datatables:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
PS C:\xampp\htdocs\Practice\Laravel\laravel_10_0_sample> composer require mediconesystems/livewire-datatables

`

How can I Solve this issue?

**Comment by joshhanley** [comment]
@maya1987 you've posted this question on Livewire's repo, but that package isn't maintained by the Livewire team. Can I suggest asking your question on their repo instead? Thanks!
